playbooks/move_vm_from_on_prem_to_aws/README.md

```diff
@@ -0,0 +1,92 @@
+# cloud.aws_ops.move_vm_from_on_prem_to_aws playbooks
+
+A playbook to migrate an existing on prem VM running on KVM hypervisor to AWS.
+
+## Requirements
+
+**qemu** and **qemu-img** packages installed.
+
+The ``cloud.aws_ops.clone_one_prem_vm`` requires privilege escalation because the .qcow2 file created by ``virt-clone`` is owned by root and ``qemu-img convert`` requires access to convert it to .raw.
```

Comment:
```suggestion
The ``cloud.aws_ops.clone_on_prem_vm`` requires privilege escalation because the .qcow2 file created by ``virt-clone`` is owned by root and ``qemu-img convert`` requires access to convert it to .raw.
```